### PR TITLE
Add Coin98 Wallet

### PR DESCRIPTION
**Pull request by coin98**
[ ] Before opening a pull request, please read the [contributing guidelines](https://github.com/pancakeswap/pancake-uikit/blob/master/CONTRIBUTING.md) first
[ ] If your PR is work in progress, open it as `draft`
[ ] Before requesting a review, all the checks need to pass
[ ] Explain what your PR does

### CODE REVIEW

**Comment by hachiojidev** [comment]
@coin98 Thank you for submitting this. We are currently discussing what path we want to take with regards additional injected wallets. For now we will close this but if we add support for additional wallets we please resubmit the relevant PR.

**Comment by Chef-Chungus** [comment]
Miscommunication on my side - reopening

File: packages/pancake-uikit/src/widgets/WalletModal/config.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import TokenPocket from "./icons/TokenPocket";
 import TrustWallet from "./icons/TrustWallet";
 import WalletConnect from "./icons/WalletConnect";
 import BinanceChain from "./icons/BinanceChain";
+import Coin98Wallet from "./icons/Coin98Wallet";
```
Comment (Chef-Chungus) [inline packages/pancake-uikit/src/widgets/WalletModal/config.tsx:7]: @coin98 - please add the Icon

**Review by hachiojidev — APPROVED** [review]

**Comment by hachiojidev** [comment]
@coin98 Please fix the commit message, tests and formatting and we can get this merged.

File: packages/pancake-uikit/src/widgets/WalletModal/icons/Coin98Wallet.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Svg from "../../../components/Svg/Svg";
+import { SvgProps } from "../../../components/Svg/types";
+
+const Icon: React.FC<SvgProps> = (props) => {
+  return (
+    <Svg
+      data-name="Layer 1"
+      xmlns="http://www.w3.org/2000/svg"
```
Comment (RabbitDoge) [inline packages/pancake-uikit/src/widgets/WalletModal/icons/Coin98Wallet.tsx:9]: please remove `xmlns` , it's part of the `Svg` component, and remove `data-name` meta data

**Review by RabbitDoge — CHANGES_REQUESTED** [review]
run `yarn test -u` for the test

File: packages/pancake-uikit/src/widgets/WalletModal/icons/Coin98Wallet.tsx (hunk 2 above)
Comment (coin98) [reply packages/pancake-uikit/src/widgets/WalletModal/icons/Coin98Wallet.tsx]: I've just resolved this. Thank you

**Comment by hachiojidev** [comment]
@coin98 You will also need to run `yarn format:write` to fix the formatting.

**Comment by coin98** [comment]
> @coin98 You will also need to run `yarn format:write` to fix the formatting.

feat: Add Coin98 Wallet
fix: Run format write
chore: Run Tests

**Comment by hachiojidev** [comment]
@coin98 The tests are still not passing. Make sure to use `yarn test -u` so the snapshots update.

**Comment by coin98** [comment]
> @coin98 The tests are still not passing. Make sure to use `yarn test -u` so the snapshots update.

feat: Add Coin98 Wallet
fix: Run test -u 
chore: Run Tests

**Pull request merged**